packages/api/cms-api/src/user-permissions/user-permissions.types.ts

```diff
@@ -73,4 +74,9 @@ export interface UserPermissionsModuleAsyncOptions extends Pick<ModuleMetadata,
     useClass?: Type<UserPermissionsOptionsFactory>;
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     useFactory?: (...args: any[]) => Promise<UserPermissionsAsyncOptions> | UserPermissionsAsyncOptions;
+    AppPermission?: Record<string, string>;
 }
+
+export const CombinedPermission: Record<string, string> = {
```

Comment:
Maybe this could be typed as `Record<Permission, Permission>`. 🤔

Reply:
all of this new types should be from type Permission instead of string 🙏

Reply:
I tried this, but this isn't possible because AppPermission would then have to include all core permissions as well. So I believe `Record<string, string>` is more fitting.